core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>Javascript Not Enabled</h2>
```

Comment:
I'm happy with that as well. 

> On May 19, 2016, at 11:05 PM, Sean Lip <notifications@github.com> wrote:
> 
> In core/templates/dev/head/base.html:
> 
> > @@ -280,6 +281,16 @@
> >                {{ warnings_and_loader() }}
> >              </div>
> >  
> > +            <noscript>
> > +              <div class="oppia-page-cards-container">
> > +                <div class="md-default-theme oppia-page-card oppia-long-text">
> > +                  <h2>Javascript Not Enabled</h2>
> Btw, something I didn't notice earlier: it's actually JavaScript, not Javascript.
> 
> I think "Your Javascript" sounds a little weird -- I don't really know what 'my JavaScript' means. What about "Please enable JavaScript in order to use Oppia", or something like that? The original header does sound a bit aloof.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>